src/hosted-buttons/utils.js

```diff
@@ -289,41 +327,96 @@ export function getButtonColor(
   };
 
   return colorMap[color][fundingSource];
-}
-
-export function shouldRenderSDKButtons(
-  fundingSources: $ReadOnlyArray<FundingSources>
-): boolean {
-  return Boolean(fundingSources.length);
-}
+};
 
-export function appendButtonContainer({
+export const applyContainerStyles = ({
   flexDirection,
-  selector,
-}: BuildButtonContainerArgs) {
-  const elm = getElementFromSelector(selector);
+  buttonContainerId,
+}: ApplyButtonStylesProps): void | Error => {
+  const buttonContainer = document.querySelector(`#${buttonContainerId}`);
 
-  if (!elm) {
-    throw new Error("PayPal button container selector was not found");
+  if (!buttonContainer) {
+    throw new Error(`Button container with id ${buttonContainerId} not found.`);
   }
 
-  const buttonContainer = document.createElement("div");
+  buttonContainer.style.cssText = `display: flex; flex-wrap: nowrap; gap: 16px; max-width: 750px; flex-direction: ${flexDirection}`;
```

Comment:
I'm not sure where we are in talks with the styles of the HTML we'll receive from the NCPS team, but I would be down to have them set all these styles on the button container except the `flex-direction`. We can handle that since it's based on the `layout` property. This way they have full control over the styles of their rendered content.

Reply:
Love that idea. 🚀

Reply:
Cool, they are good with setting these styles. I'll change this to:
```
buttonContainer.style.flexDirection = flexDirection;
```